src/components/compound/DlSelect/DlSelect.vue

```diff
@@ -119,9 +119,8 @@
                                 (!multiselect && searchable && !isExpanded)
                         "
                         class="selected-label"
-                    >
-                        {{ getLabel(selectedOption) }}
-                    </span>
+                        v-html="getLabel(selectedOption)"
```

Comment:
Same here, why do oyu need to inject html ?

Reply:
@guyDataloop okay, copying from slack:
> I’ve found that DlSelect drop down menu does treat option.label as html (for searchable mode support), so just by doing the same while displaying selected option we can provide custom html for options display everywhere in DlSelect. this seems like the least invasive fix